app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -21,6 +23,7 @@ import DeviceContext from '../context/device';
  * @prop {number=} minimumSharpnessScore Minimum sharpness score to be considered acceptable.
  * @prop {number=} minimumFileSize Minimum file size (in bytes) to be considered acceptable.
  * @prop {boolean=} allowUpload Whether to allow file upload. Defaults to `true`.
+ * @prop {ReactNode=} error Error to show.
```

Comment:
> I guess we have the types to annotate this...but my guess without looking here is that `error` would be an instance of `Error`, not the formatted error message as a `ReactNode` .... would it make sense to call this `errorMessage` or `errorNode`?

Yeah, I agree that it seems like it could be a source of confusion 👍 

There's a lot of existing references to `error` as messages already in `AcuantCapture` and `FileInput` which should probably be brought into alignment if we're to change it.

Maybe `error: string` didn't seem quite as problematic since `string` is a primitive type, whereas `ReactNode` is more prone to being confused as "is it an instance of an `Error`"?

Reply:
In e1bf589, references to "error" in the context of being a visual "error message" where renamed to better represent this distinction between error objects and error messages.